### DISCUSSION TITLE
New "About Cuppa" page

### DISCUSSION THREAD

**Discussion by ncosgray** [ncosgray/cuppa_mobile#68] [Announcements]
Hi everyone!

I've added a new "About Cuppa" page for the next release of Cuppa. All of the items on this screen are simply links to the relevant GitHub and Weblate pages. Previously, Cuppa only had the small "About"-style text box at the bottom of the Prefs page, linking to my website, so this new page adds a lot more info up front. I hope it is useful. I think this will be a convenient way for people to learn more about the Cuppa project and even get involved if they like.

See anything missing from this page? If you have feedback, please reply below with your ideas.

And you can help translate it to other languages here: https://hosted.weblate.org/projects/cuppa/cuppa-mobile-app/

![image](https://user-images.githubusercontent.com/13181064/152223381-722f4dc6-6112-490e-b7d0-30551d6b382d.png)

**Comment by DesmoricFR** [comment]
I suggest to add an "how to use" page. I know that Cuppa is an easy to use application, but it woul be nice to write just new lines of documentation to explain the principles of use. I think, in order :
- add a new favorite the,
- how to run the first timer
- how to run the second timer

So, I imagine a very simple description, with 3 blocks of text about 3 or 4 lines, no more.

I can do that in French (my native language) and in english. Then, you could add the page and associate the right text with the prefered language.

Do you consider this idea as good ?

**Reply by DesmoricFR** [reply]
It can be very simple, as this :

In english

1- go in parameters (cogwheel)
2- Clic on "Add an infusion"
* set a name
* set duration
* set temperature
* choose a colour (optional)
3- clic on the infusion to start the first count-down
4- clic on a second infusion to start a second count-down

In French
1- Aller dans les paramètre (roue crantée)
2- Cliquer sur « Ajouter une infusion »
* donner un nom à cette infusion
* régler la durée
* régler la température
* choiser la couleur (optionnel)
3- Appuyer sur l’infusion concernée pour lancer le compte à rebours
4- Appuyer sur une seconde infusion pour lancer un deuxième compte à rebours

**Reply by ncosgray** [reply]
Thank you so much for this input! In the next version, there will be an onboarding guide for new users that briefly explains how the app works. It will be shown only once **for brand-new installations of Cuppa** (if you already have the app installed, you won't see it!) but the tutorial will also be accessible from the About screen.

You can see what the new tutorial will look like and contribute translations of the text in Weblate: https://hosted.weblate.org/translate/cuppa/cuppa-mobile-app/en/?&offset=52

**Reply by DesmoricFR** [reply]
You're welcome 😃. I use your app, so I décided to contribute as I can. 
I did french translation as you suggested, on weblate 😉.
